lib/ansible/modules/cloud/azure/azure_rm_rediscache.py

```diff
@@ -561,7 +562,7 @@ def create_rediscache(self):
             response = self._client.redis.create(resource_group_name=self.resource_group,
                                                  name=self.name,
                                                  parameters=params)
-            if isinstance(response, AzureOperationPoller):
+            if isinstance(response, LROPoller) or isinstance(response, AzureOperationPoller):
```

Comment:
why there will be 2 returned response? is it caused by sdk version difference?

Reply:
AzureOperationPoller is older versions of Python SDK< while LROPoller is in newer versions.
In some cases.
Some operations may return poller randomly (depending on some circumstances), so sometimes upgrading package version may leave poller issue undetected by CI.
Then it may manifest itself randomly in the future.
So it's better to be safe and be prepared.